src/containers/BlogCard/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import { Row, Col, Image } from "antd";
+import "./index.css";
+import LoadingPage from "../../components/LoadingPage";
+
+const Story = () => {
+  const [blog, setBlog] = React.useState();
+  let params = useParams();
+  const storyID = params.id;
+
+  React.useEffect(() => {
+    const _storyURL = `https://public-api.wordpress.com/wp/v2/sites/feedtheneedy518055725.wordpress.com/posts/${storyID}`;
+    const getBlog = async () => {
+      const data = await fetch(_storyURL);
+      const fetchedStory = await data.json();
+      setBlog(fetchedStory);
+    };
+    getBlog();
+  }, [storyID]);
+
+  if (!blog) {
+    return <LoadingPage />;
+  } else {
+    return (
+      <div className="blogContainer">
+        <Row>
+          <Col>
+            <h1
+              className="blogTitle"
+              dangerouslySetInnerHTML={{
```

Comment:
Why are you using innerHTML here? That should be avoided.

Reply:
@bertacsevercan this needs to be used for the blogs